MARDEK Engine/Assets/Scripts/MARDEK/Inventory/Item/UnobtainedPlotItems.cs

```diff
@@ -0,0 +1,10 @@
+using UnityEngine;
+
+namespace MARDEK.Inventory{
+    [System.Serializable]
+    public class UnobtainedPlotItems
```

Comment:
This could be named PlotItemsToImport

Reply:
This one is also a class I ended up not using and deleted, trying to figure out how it got in the pull request.